waddrmgr/db.go

```diff
@@ -1866,6 +1870,17 @@ func putSyncedTo(ns walletdb.ReadWriteBucket, bs *BlockStamp) error {
 	if err != nil {
 		return managerError(ErrDatabase, errStr, err)
 	}
+
+	// Finally we delete any hashes for heights below our max reorg depth.
+	forgetHeight := bs.Height - maxReorgDepth
+	if forgetHeight > 0 {
+		height := make([]byte, 4)
+		binary.BigEndian.PutUint32(height, uint32(forgetHeight))
+		if err := bucket.Delete(height); err != nil {
```

Comment:
One thing we should consider is for user having old databases, this will not delete the oldest hashes. We could consider looping up to this height to delete them.

Reply:
I think we can add that in a follow up, as fwiw, this is mainly for new nodes coming up which will be our application and btcd/bitcoind nodes.

Reply:
```suggestion
		if err := bucket.Delete(height); err != nil {
```